windows-imager.zig

```diff
@@ -403,6 +404,68 @@ pub fn main2() anyerror!u8 {
         return 0;
     }
 
+    if (mem.eql(u8, cmd, "read")) {
```

Comment:
I like having the simpler commands first, I think it makes it a little easier to follow what's going on.  So this could go before or after we handle the `"image"` command.